packages/ERTP/src/payment.js

```diff
@@ -1,15 +1,21 @@
 // @ts-check
 
-import { Far } from '@endo/marshal';
+import { defineKind } from '@agoric/swingset-vat/src/storeModule.js';
```

Comment:
this deep import is unfortunate. I'll prioritize https://github.com/Agoric/agoric-sdk/issues/4255

Reply:
Thanks! See also my comment https://github.com/Agoric/agoric-sdk/pull/4618#issuecomment-1074462354 below.